src/main/java/org/geppetto/frontend/controllers/ConnectionHandler.java

```diff
@@ -190,6 +193,10 @@ public void loadGeppettoProject(String requestID, IGeppettoProject geppettoProje
 			setConnectionProject(geppettoProject);
 			websocketConnection.sendMessage(requestID, OutboundMessages.PROJECT_LOADED, projectJSON);
 
+			boolean persisted = !geppettoProject.isVolatile();
+			String update = "{\"persisted\":" + persisted + ",\"projectId\":" + geppettoProject.getId()  +"}";
```

Comment:
@slarson That message template is on the javascript side to send outgoing messages, no use in this java class.